api/base/generic_bulk_views.py

```diff
@@ -115,6 +125,18 @@ def bulk_destroy(self, request, *args, **kwargs):
         if not self.allow_bulk_destroy_resources(user, resource_object_list):
             raise PermissionDenied
 
-        self.perform_bulk_destroy(resource_object_list)
+        # If skip_uneditable=True in query_params, skip resources for which the user does not have admin permissions
+        # and delete the remaining resources
+        if self.request.query_params.get('skip_uneditable', 'False') == 'True':
+            allowed = []
+            skipped = []
+            for resource in resource_object_list:
+                if resource.has_permission(user, ADMIN):
+                    allowed.append(resource)
+                else:
+                    skipped.append({'id': resource._id, 'type': object_type})
+            self.perform_bulk_destroy(allowed)
+            return Response(status=status.HTTP_200_OK, data={'meta': {'errors': skipped}})
```

Comment:
Please also add tests to confirm that the proper method is returned here, after the above change goes in.